components/camera/server.go

```diff
@@ -89,9 +90,8 @@ func (s *subtypeServer) GetImage(
 		}
 	}()
 
-	actualMIME, _ := utils.CheckLazyMIMEType(req.MimeType)
 	resp := pb.GetImageResponse{
-		MimeType: actualMIME,
+		MimeType: req.MimeType,
```

Comment:
This doesn't make much sense to return. A gRPC response isn't lazy; the data is there and lazy indicates a hint as to how we decode intra-process. I'd keep these few lines as is but keep 82.

Reply:
If that's the case, then `img, err := rimage.DecodeImage(ctx, resp.Image, resp.MimeType)` in client.go needs to change. Rather than feeding the `resp.MimeType` directly in, wrap it with +lazy first.

Reply:
yeah feeding in lazy to Decode makes sense. That describes the intent of what to do to that.

Reply:
Makes sense. Done